iphone/cli/commands/_build.js

```diff
@@ -4092,6 +4096,11 @@ iOSBuilder.prototype.writeInfoPlist = function writeInfoPlist() {
 		delete plist.UILaunchImageFile;
 	}
 
+	// inject method if we are building for dist-appstore
+	if (this.target === "dist-appstore") {
+		plist.method = "app-store";
```

Comment:
I'll remove this, we'll be doing this anyway on the new centralized export ipa function that would be used both for ad-hoc and appstore builds.